src/transformers/models/gpt_neox/modeling_gpt_neox.py

```diff
@@ -1025,22 +1137,42 @@ def forward(
         )
 
     def prepare_inputs_for_generation(
```

Comment:
seems to be the same as llama: no token type ids

Reply:
seems like yes, will check and add copy statement if possible